### ISSUE TITLE
about the nerf training code

### ISSUE THREAD

**Issue by MingYi410** [nerf2nerf/nerf2nerf#8]
Hello,
thank you for sharing your excellent work. I'm puzzled by how the nerf model is trained when reading your code, could you please sharing your nerf training code as well? Thank you very much.
Good Luck!

**Comment by lilygoli** [comment]
you can find the link to NeRF training code in #6

**Comment by MingYi410** [comment]
Thanks for your reply, the code helps me a lot. But I’m still a little confused about how to train the auxiliary transmittance_field network.  In my understanding, the `fine_sample_transmittance` is the gt value of  `predicted_fine_transmittance`, and I should define the transmittance training loss by comparing these two components, right ?

**Comment by lilygoli** [comment]
Yes, that is correct, you can simultaneously train the auxiliary transmittance field when training NeRF by supervising the auxiliary field with the volume rendering values (density and then transmittance) you get for sample points on the rays.
